components/dashboard/sections/overview/TimelineItem.tsx

```diff
@@ -18,52 +18,92 @@ import { Skeleton } from '../../../ui/Skeleton';
+    <div className="mt-4 space-y-2 rounded-xl bg-slate-50  p-4">
+      {title && <p className="text-base font-medium">{title}</p>}
+      <HTMLContent fontSize="14px" lineHeight="20px" content={content} />
+      {summary && (
+        <div>
+          <StyledLink
+            as={Link}
+            fontSize="14px"
+            lineHeight="16px"
+            href={href}
+            className="font-medium !text-slate-900 hover:underline"
+          >
+            <FormattedMessage id="ContributeCard.ReadMore" defaultMessage="Read more" />
+          </StyledLink>
+        </div>
+      )}
+    </div>
+  );
+};
+
 const TimelineItem = ({ activity, openExpense }: ActivityListItemProps) => {
   const intl = useIntl();
-  const secondaryAccount = activity?.individual?.id !== activity?.account?.id && activity.account;
-  const content = activity?.data?.comment?.html || activity?.update?.summary;
 
   const isLoading = !activity;
   const isLastWeek = dayjs(activity?.createdAt).isAfter(dayjs().subtract(1, 'week'));
 
-  const displayFollowButton = activity?.type === ActivityType.COLLECTIVE_UPDATE_PUBLISHED;
-  const i18nMsg = ActivityTimelineMessageI18n[activity?.type] || ActivityDescriptionI18n[activity?.type];
-
+  const timelineMessage = ActivityTimelineMessageI18n[activity?.type];
+  const i18nMsg = timelineMessage || ActivityDescriptionI18n[activity?.type];
+  const hasTimeLineMessage = Boolean(timelineMessage);
   let description = null;
   if (i18nMsg) {
-    description = intl.formatMessage(
-      i18nMsg,
-      getActivityVariables(intl, activity, { onClickExpense: openExpense, displayFollowButton }),
-    );
+    description = intl.formatMessage(i18nMsg, getActivityVariables(intl, activity, { onClickExpense: openExpense }));
   } else {
     description = capitalize(activity?.type.replace('_', ' '));
   }
 
+  // Pick which account to use as the avatar
+  let avatar = 'individual';
+  if (i18nMsg?.avatar) {
+    avatar = i18nMsg.avatar;
+  }
+
+  // Pick which content card to use
+  const contentKey = i18nMsg?.content;
+
   return (
     <div className="rounded-2xl border p-4 text-sm">
       <div className="flex flex-1 items-start gap-3">
         {isLoading ? (
           <Skeleton className="h-8 w-8 shrink-0 rounded-full" />
         ) : (
-          <AvatarWithLink
-            size={32}
-            account={activity.individual || activity.fromAccount}
-            secondaryAccount={secondaryAccount}
-          />
+          hasTimeLineMessage && <AvatarWithLink size={32} account={activity[avatar]} />
         )}
         {isLoading ? (
           <Skeleton className="h-4 w-80" />
         ) : (
-          <div className="flex min-w-0 flex-1 flex-wrap items-center justify-between gap-1">
-            <div className="text-foreground">{description}</div>
+          <div className="flex min-w-0 flex-1 flex-wrap  items-center justify-between gap-1 pt-1">
+            <div className="leading-6 text-foreground">{description}</div>
```

Comment:
I think the way we wrap this looks a bit weird. Perhaps we should wrap only the description and not the date?
![image](https://github.com/opencollective/opencollective-frontend/assets/2119706/faadf7fc-c3b2-410a-920f-20c6aa7cc494)